src/Fuzz.elm

```diff
@@ -241,6 +241,14 @@ string =
         Shrink.string
 
 
+{-| A fuzzer for string values. Generates random printable ascii strings whose
+length can be specified.
+-}
+longString : Int -> Fuzzer String
+longString maxLen=
```

Comment:
I think `longString` is a bad name for this function. It would also be a good idea to make it `Int -> Int -> Fuzzer String`, so that both the smallest and longest string can be set